### PR TITLE
A new post about custom 3D engine

### PR DESCRIPTION
**Pull request by rougier**
A new blog post about a custom 3D engine for Matplotlib. I tried to follow the tutorial on creating blog post. Hope everything's fine.

### CODE REVIEW

**Review by story645 — APPROVED** [review]
This is really great, thanks for submitting it!

File: content/posts/custom-3d-engine/index.md [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+
+For the perspective projection, we also need to specify the aperture angle that
+(more or less) sets the size of the near plane relatively to the far
+plane. Consequently, for high apertures, you'll get a lot of "deformations".
+
+However, if you look at the two functions above, you'll realize they return 4x4
+matrices while our coordinates are 3d. How to use these matrices then ? The
+answer is [homogeneous
+coordinates](https://en.wikipedia.org/wiki/Homogeneous_coordinates). To make
+along story short, homogeneous coordinates are best to deal with transformation
```
Comment (story645) [inline content/posts/custom-3d-engine/index.md]: Typo a long

**Comment by rougier** [comment]
I've done some corrections.

**Comment by davidevaleriani** [comment]
Great post, thank you @rougier!

**Pull request merged**